lib/sequent/core/helpers/default_validators.rb

```diff
@@ -3,9 +3,18 @@ module Core
     module Helpers
       class DefaultValidators
         VALIDATORS = {
-          Integer => ->(klass, field) { klass.validates_numericality_of field, only_integer: true, allow_nil: true, allow_blank: true },
-          Date => ->(klass, field) { klass.validates field, "sequent::Core::Helpers::Date" => true },
-          DateTime => ->(klass, field) { klass.validates field, "sequent::Core::Helpers::DateTime" => true }
+          Integer => ->(klass, field) do
+            klass.validates_numericality_of field, only_integer: true, allow_nil: true, allow_blank: true
+          end,
+          Date => ->(klass, field) do
+            klass.validates field, "sequent::Core::Helpers::Date" => true
```

Comment:
Moet Sequent hier niet met een hoofdletter? (x3)

Reply:
That is how these validators work in active validation.